.github/workflows/autobuild.yml

```diff
@@ -246,79 +233,83 @@ jobs:
-          if-no-files-found:        error # 'warn' or 'ignore' are also available, defaults to `warn`
-
-      # Notarize macOS build, if needed
-      - name: "Notarize macOS Release Build"
-        if: ${{ (steps.step_cmd3_postbuild.outputs.artifact_1 != '') && (steps.step_macos_build.outputs.macos_signed == 'true') && contains(needs.create_release.outputs.publish_to_release, 'true') }}
-        id: notarize-macOS-app
-        uses: devbotsxyz/xcode-notarize@d7219e1c390b47db8bab0f6b4fc1e3b7943e4b3b
+          if-no-files-found:        error
+
+      - name:                       Notarize macOS Release Build
+        if:                         >-
+                                    steps.step_cmd3_postbuild.outputs.artifact_1 != '' &&
+                                    steps.step_build.outputs.macos_signed == 'true' &&
+                                    contains(needs.create_release.outputs.publish_to_release, 'true')
+        id:                         notarize-macOS-app
+        uses:                       devbotsxyz/xcode-notarize@d7219e1c390b47db8bab0f6b4fc1e3b7943e4b3b
         with:
-          product-path: deploy/${{ steps.step_cmd3_postbuild.outputs.artifact_1 }}
-          primary-bundle-id: io.jamulus.Jamulus
+          product-path:             deploy/${{ steps.step_cmd3_postbuild.outputs.artifact_1 }}
+          primary-bundle-id:        io.jamulus.Jamulus
           appstore-connect-username: ${{ secrets.NOTARIZATION_USERNAME }}
           appstore-connect-password: ${{ secrets.NOTARIZATION_PASSWORD }}
-      # Apply successful notarization
-      - name: "Staple macOS Release Build"
-        if: ${{ (steps.step_cmd3_postbuild.outputs.artifact_1 != '') && (steps.step_macos_build.outputs.macos_signed == 'true') && contains(needs.create_release.outputs.publish_to_release, 'true') }}
-        id: staple-macOS-app
-        uses: devbotsxyz/xcode-staple@v1
+
+      - name:                       Staple macOS Release Build
+        if:                         >-
+                                    steps.step_cmd3_postbuild.outputs.artifact_1 != '' &&
+                                    steps.step_build.outputs.macos_signed == 'true' &&
+                                    contains(needs.create_release.outputs.publish_to_release, 'true')
+        id:                         staple-macOS-app
+        uses:                       devbotsxyz/xcode-staple@v1
         with:
-          product-path: deploy/${{ steps.step_cmd3_postbuild.outputs.artifact_1 }}
+          product-path:             deploy/${{ steps.step_cmd3_postbuild.outputs.artifact_1 }}
 
-      # Upload Artifact to Release
-      - name:                       Upload Artifact 1 to Release
-        if:                         ${{ (steps.step_cmd3_postbuild.outputs.artifact_1 != '') && contains(needs.create_release.outputs.publish_to_release, 'true') }}
+      - name:                       Upload Artifact 1 to Release if this is a release build
```

Comment:
Not sure the "if this is a release" is needed. If it's not a release, the step will be shown in the log as not executed anyway. Same for artifact 2 below.

Reply:
I might have added this in response to feedback as well, but I've lost track by now...
@ann0see Should I shorten it as well? :)

Reply:
Probably it’s redundant so yes, you can remove it.

Reply:
Done. Will merge on CI green.